src/test/java/org/prebid/server/handler/VtrackHandlerTest.java

```diff
@@ -94,10 +98,25 @@ public void shouldRespondWithBadRequestWhenAccountParameterIsMissing() {
         verify(httpResponse).end(eq("Account 'a' is required query parameter and can't be empty"));
     }
 
+    @Test
+    public void contentTypeHeaderAdded() {
```

Comment:
Test method must be in style: `shouldRespondWithExpectedHeaders`.
Please see existing tests as example.